src/components/GameDetail/index.jsx

```diff
@@ -49,67 +49,72 @@ export default function GameDetail() {
     <div className="game-detail">
       <h1 className="game-title">{name || <Skeleton width="400px" />}</h1>
       <div className="game-detail-container">
-        <div className="game-detail-left">
+        <div className="game-content">
           {!loading ? (
             <img className="game-image" src={background_image} alt="game" />
           ) : (
-            <Skeleton height="400px" width="53vw" />
+            <Skeleton style={{ aspectRatio: "16/9" }} />
           )}
-          <div className="game-description">
-            <h2>About this Game</h2>
+          <div className="game-info">
             <div>
-              {!loading ? (
-                description ? (
-                  parse(`${description}`)
+              <p className="subtitle">Genre: </p>
+              <p className="subtitle-data">
+                {!loading ? (
```

Comment:
Vinayak instead of doing loading check in each line you can have a top level loading check and then return loading or content based on the loading value